### PR TITLE
#164691204 Upgrade project to django 2.1

### PR DESCRIPTION
**Pull request by Manorlds-Eaglespark**
#### What does this PR do?
- ensure the project shell is using the latest version of django at the time of development

#### Description of Task to be completed?
- Upgrading the project base code to Django 2.1

#### Any background context you want to provide?
Django 2.x is the latest release of Django with full support of Python 3.x series.

#### What are the relevant pivotal tracker stories?

[#164691204](https://www.pivotaltracker.com/n/projects/2319850)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -82,6 +82,7 @@ celerybeat-schedule
 # virtualenv
 venv/
 ENV/
+myenv/
```
Comment (NicholusMuwonge) [inline .gitignore:85]: Is this an environment created by one of the team mates or, just a default django env

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
-Django==1.10
```
Comment (NicholusMuwonge) [inline requirements.txt:1]: I see you finally added the requirements.txt back, thanks for the good work. I think my concerns from yesterday are now settled.

**Review by NicholusMuwonge — APPROVED** [review]
I think all is well now, according to my concerns from yesterday. Thanks for looking into them, I think its ready to merger

File: authors/urls.py [hunk 3]
```diff
@@ -15,10 +15,11 @@
 """
 from django.conf.urls import include, url
```
Comment (AmosWels) [inline authors/urls.py:16]: You can remove this import of `url`

File: authors/apps/authentication/urls.py [hunk 4]
```diff
@@ -1,11 +1,11 @@
 from django.conf.urls import url
```
Comment (AmosWels) [inline authors/apps/authentication/urls.py:1]: This import is unused since you now use `path` and not `url`, please remove the import

**Review by AmosWels — CHANGES_REQUESTED** [review]
Good work @Manorlds-Eaglespark, update your commit message using the recommended team conventions that are found here https://github.com/aqua-organisation/ah-django-Aqua/wiki/Coding-Conventions-for-ah-django-Aqua

**Pull request merged**